src/main/java/org/team1540/robot2025/subsystems/grabber/GrabberIO.java

```diff
@@ -0,0 +1,23 @@
+package org.team1540.robot2025.subsystems.grabber;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface GrabberIO {
+    @AutoLog
+    class GrabberIOInputs {
+        public double motorCurrentAmps = 0.0;
+        public double motorAppliedVolts = 0.0;
+        public double motorTempCelsius = 0.0;
+        public double velocityRPM = 0.0;
+        public double positionRots = 0.0;
+        public boolean hasCoral = false;
+    }
+
+    default void updateInputs(GrabberIOInputs inputs) {}
+
+    default void setVoltage(double volts) {}
+
+    default void setVelocity(double velocityRPM) {}
```

Comment:
I don't think we'd really ever need to do a velocity closed-loop on the grabber, open loop voltage should work fine.